packages/py-moose-lib/README.md

```diff
@@ -1,3 +1,72 @@
 # Python Moose Lib
 
-Python package which contains moose utils
+Python package which contains moose utils and streaming functionality.
```

Comment:
I'm concerned about maintaining these docs and the ones on https://docs.getmoose.dev. Should we pick one?